### ISSUE TITLE
Some user visit/actions are not tracked in piwik

### ISSUE THREAD

**Issue by BrijeshSetty** [piwik/piwik#12078]
Sometimes we see few users are not tracked. Piwik able to track those users for few days & later sometime it missed to update in database. In rest response piwik says 200 OK successful in response but it is actually not updated database, it is missed to update.

Ex:
Piwik was able to track & update the events for the other/same users before & after that 1 mins, only issue occur for sometimes.

I

**Comment by sgiehl** [comment]
There are many reasons why visitors might not be tracked (like AdBlocker, DNT, Opt-Out,...).
Can you give any more detailed information that would allow us to reproduce your issue in any way?

**Comment by BrijeshSetty** [comment]
We have analysed on Adblocker, DNT, Opt-Out. But this issue is not on the browser side. 
From browser piwik request is going to Piwik server & Piwik server response says 200 OK. For the same user piwik able to track for 10 mins before & 10 mins after, but problem is only for that 10 mins.

We investigated http server logs & could see in that 10 mins duration server logs are captured & data is not pushed to DB. When we did replay of those users for those action then piwik able to capture data in DB.

This is inconsistent issue that happens only for some duration & works fine later.

**Comment by mattab** [comment]
This sounds like a server issue rather than Piwik issue. Is there any error in the web server error log?

**Comment by BrijeshSetty** [comment]
10.0.3.141 - - [30/Aug/2017:13:31:37 +0000] "GET /piwik/piwik.php?action_name=MyApp&idsite=1&rec=1&r=661663&h=9&m=32&s=28&url=https%3A%2F%2Fmyapp.com%2F&uid=abc@myapp.com&_id=dda6c4dce8eefff6&_idts=1504099943&_idvc=1&_idn=0&_refts=0&_viewts=1504099943&send_image=1&pdf=0&qt=0&realp=0&wma=0&dir=0&fla=1&java=1&gears=0&ag=1&cookie=1&res=1280x1024&gt_ms=8 HTTP/1.1" 200 86 "https://myapp.com/" "Mozilla/5.0 (Windows NT 6.1; WOW64; Trident/7.0; rv:11.0) like Gecko" 

This is server logs. Piwik said 200 OK but not updated in database

**Comment by BrijeshSetty** [comment]
There is NO error trace either in Web server or Piwik :(

**Comment by sgiehl** [comment]
How is your website configured in Piwik? Have you set all urls for the website? Is the option to only track visits for those urls active?

**Comment by BrijeshSetty** [comment]
Yes it is configured & all page of mypp page we are sending calls to piwik. 

Note: This issue is happening only for few minutes & again it is tracked user visits. It is only happening for 4-5 mintues duration

**Comment by sgiehl** [comment]
How is your server set up? Do you use a proxy?

**Comment by BrijeshSetty** [comment]
Piwik & Myapp hosted on same server. We are not using proxy.

**Comment by BrijeshSetty** [comment]
Piwik & Myapp deployed on same AWS server. We are not using proxy.

**Comment by BrijeshSetty** [comment]
One more point when we do replay for missed users 
then it will successfully update in database

**Comment by mattab** [comment]
It sounds like a server issue rather than Piwik issue
